### PR TITLE
Cleaner jenkins

### PR DESCRIPTION
**Pull request by acpaquette**
Cleaning up most of the failing jenkins tests

## Description
Cleans up a majority of the failing tests on Jenkins leaving 4 module tests still failing and 1 app test failing. The four module tests are:
948 - hayabusa_module_test_fullframe (Failed) - 1e-12 difference in some angle data
949 - hayabusa_module_test_subframe (Failed) - 1e-12 difference in some angle data
1102 - mro_module_test_hirise (Failed) - Currently failing on linux unknown as to why
1126 - osirisrex_module_test_mapcam (Failed) - 1e-12 difference in some angle data

The one failing app test is:
1029 - mdisedrinfo_app_test_kernelchk (Failed) - 1e-12 difference in some angle data

## Related Issue
N/A

## How Has This Been Validated?
These are changes to tests

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Infrastructure change (changes to things like CI or the build system that do not impact users)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [ ] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

**Comment by acpaquette** [comment]
Final old app test to update is `mdisedrinfo` there seem to be updates to the spice data so I can't really debug anything if it is related to the camera.cpp changes from #4980

**Comment by acpaquette** [comment]
Latest build results located here:
https://jenkins.prod-asc.chs.usgs.gov:8443/job/ISIS/view/change-requests/job/PR-5081/

File: isis/src/base/apps/getsn/getsn.cpp [hunk 1]
```diff
@@ -89,6 +89,12 @@ namespace Isis {
       }
     }
 
-    log->addLogGroup(sn);    
+    // log->addLogGroup(sn);
```
Comment (amystamile-usgs) [inline isis/src/base/apps/getsn/getsn.cpp]: Do we need to keep this comment?

**Review by amystamile-usgs — DISMISSED** [review]
One comment. Otherwise looks good.

File: isis/src/base/apps/getsn/getsn.cpp (hunk 1 above)
Comment (acpaquette) [reply isis/src/base/apps/getsn/getsn.cpp]: No, it should be removed

**Review by amystamile-usgs — APPROVED** [review]

**Pull request merged**